docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -301,6 +326,24 @@ The following code registers the converter:
 
 [!code-csharp[](snippets/system-text-json-how-to/csharp/RoundtripStackOfT.cs?name=SnippetRegister)]
 
+## Handle null values
+
+By default, the serializer does not pass `null` to custom converters for reference types and primitive value types on serialization and deserialization:
+
+* For reference types, it returns a `null` instance on deserialization.
+* For primitive value types, it throws a `JsonException` or returns a default value, depending on if there's an internal converter.
+* It writes `null` directly with the writer on serialization.
+
+This behavior is primarily to optimize performance by skipping an extra call to the converter. In addition, it avoids forcing converters to check for `null` at the start of every `Read` and `Write` method override.
+
+For `Nullable<T>` value types, the serializer does pass `null` to converters because it isn't a valid CLR value for value types. This way, the converter can determine what to do with this "invalid" token.
```

Comment:
```suggestion
For  value types, the serializer passes `JsonTokenType.Null` values to converters on deserialization because it isn't a valid CLR value for value types. This way, the converter can determine what to do with this "invalid" token.
```